app/models/concerns/anonymization.rb

```diff
@@ -13,13 +13,23 @@ module Anonymization
 
   module ClassMethods
     def anonymize_petitions!(time = Time.current)
-      in_need_of_anonymizing(6.months.ago(time)).find_each do |petition|
+      in_need_of_anonymizing.find_each do |petition|
+        petition.anonymize!
+      end
+    end
+
+    def in_need_of_anonymizing(time = Time.current)
+      not_anonymized.where(do_not_anonymize: [nil, false])
```

Comment:
Since we're using it more than once, would it be worth having a class method `.do_not_anonymize`, in the same way as we have `not_anonymized`?